base/src/main/java/com/austindroids/austinfeedsme/events/EventsAdapter.kt

```diff
@@ -19,13 +19,11 @@ import com.bumptech.glide.Glide
 /**
  * Created by darrankelinske on 12/3/16.
  */
-internal class EventsAdapter(private val context: Context, Events: List<Event>,
+internal class EventsAdapter(private var context: Context, var events: List<Event>,
```

Comment:
Why is `context` a `var`? This shouldn't be a `var`.

Reply:
are saying it should be a val or we should completely remove the context reference from the adapter? Changed it to val